__test__/locale.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, it, expect } from 'vitest'
+import { resolve } from 'node:path'
+import { readdirSync } from 'node:fs'
+import { Language } from '../utils/getLanguage'
+import { includeAllKeys, excludeKeys } from './utils'
+
+const locales = readdirSync(resolve(__dirname, '../locales')).filter((file) => {
+  return file.includes('.json')
+})
+
+describe('should match name regex', () => {
```

Comment:
maybe remove this file for now, and open a dedicated PR with it, once this one is merged.
I think we can merge with `getCommand.spec.ts` and `sortDependencies.spec.ts` (so you remove this file, the schema, the `utils` file from this PR)